README.md

```diff
@@ -35,9 +155,104 @@ make dev
 
 You can now access the entry point at <http://localhost:8080>, or view the console at <http://localhost:8081>.
 
+<details>
+
+<summary>Rebuilding the container</summary>
+
+Code is attached to the containers as volumes so most updates are visible without a need to rebuild the container.
+However, changes to NPM dependencies, for example, require a rebuild. So you may need to execute
+
+```shell
+make build
+```
+
+before running further commands.
+
+</details>
+
+### Running the tests
+
+The app is tested using [Jest]. It can be run by executing: 
+
+```shell
+make test
+```
+
+The tests can also be run in separate suites:
+
+```shell
+make unit-test
+make integration-test
+```
+
+Integration tests are separated using the `@group integration` annotation, and can access a PostgreSQL instance. 
+
+### Linting
+
+The app is linted using [ESLint]. It can be run by executing:
+
+```shell
+make lint
+```
+
+Problems can be automatically fix, where possible, by executing:
+
+```shell
+make fix
+```
+
+Contributing
+------------
+
+Pull requests and other contributions are more than welcome. Please take a look at the [contributing guidelines] for
+further details.
+
 Getting help
 ------------
 
-- Report a bug or request a feature on [GitHub](https://github.com/libero/publisher/issues/new/choose).
-- Ask a question on the [Libero Community Slack](https://libero.pub/join-slack).
-- Read the [code of conduct](https://libero.pub/code-of-conduct).
+- Report a bug or request a feature on [GitHub][new issue].
+- Ask a question on the [Libero Community Slack].
+- Read the [code of conduct].
+
+License
+-------
+
+This software is released under the [MIT license][license]. Copyright © 2019 [eLife Sciences Publications, Ltd][eLife].
```

Comment:
Should be 2020 I guess

Reply:
Following @BlueReZZ's advice, it's when the repo was created (otherwise some commits aren't covered).

Reply:
Should it be a range? If so 2019-2020.

Reply:
This is more complicated than people think

Reply:
A range is also fine - some people prefer that as it shows it's not outdated, and gives you belt-and-braces to cover code that existed this year. Copyrights last for 70 years under common law and work on a  "whoever got there first" basis until you start the argument, so it's best to match the inception date of the repo.

My opinion appears to be corroborated by SO (confirmation bias FTW):
> The copyright notice on a work establishes a claim to copyright. The date on the notice establishes how far back the claim is made. This means if you update the date, you are no longer claiming the copyright for the original date and that means if somebody has copied the work in the meantime and they claim its theirs on the ground that their publishing the copy was before your claim, then it will be difficult to establish who is the originator of the work.

src: https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated